includes/class-signature.php

```diff
@@ -214,7 +214,7 @@ public static function generate_signature( $user_id, $http_method, $url, $date,
 		\openssl_sign( $signed_string, $signature, $key, \OPENSSL_ALGO_SHA256 );
 		$signature = \base64_encode( $signature ); // phpcs:ignore
 
-		$key_id = $user->get_url() . '#main-key';
```

Comment:
@jeherve @mediaformat @mattwiebe this seem to fix the pending follow request on GoToSocial. But I have no idea why. Does that makes sense?

Reply:
I am able to follow a site running this branch from my GoToSocial instance, and new posts are also pushed, so this seems to work. I do not know if this would break something else, though. :|